cypress/integration/landing_page_spec.ts

```diff
@@ -0,0 +1,64 @@
+import { LandingPo } from '../support/landing.po';
+
+describe('Landing Page test suite', () => {
+  const landing = new LandingPo();
+
+  it('Successfully loads and displays all content of the ngx-bootstrap LP', () => {
+    landing.navigateTo();
+
+    cy.get('.logo').should('be.visible');
+    cy.get('.header-info').should('be.visible');
+    cy.get('.content-logo').should('be.visible');
+    cy.get('.slogan').should('be.visible');
+    cy.get('.descr').should('be.visible');
+    cy.get('.version').should('be.visible');
+    cy.get('.advantages').should('be.visible');
+  });
+
+  it('Get started button redirects to Getting Started page', () => {
+    const buttonText = 'Get started';
+    const searchedUrl = '/getting-started';
+
+    landing.navigateTo();
+    landing.clickByText('.btn', buttonText);
+
+    cy.url().should('include', searchedUrl);
+  });
+
+  it('Github button is enabled and contains link to ngx-bootstrap repo', () => {
+    const buttonText = 'Github';
+
+    landing.navigateTo();
+
+    cy.get('.btn').contains(buttonText).should('be.enabled');
+    cy.get('.btn').contains(buttonText).should('have.attr', 'href', landing.githubUrl);
+  });
+
+  it('Info buttons in header are enabled and contains links to slack, github and stackoverflow', () => {
+    const buttonSelector = '.header-list li a';
+
+    landing.navigateTo();
+
+    cy.get(buttonSelector).eq(0).should('be.enabled');
```

Comment:
use aliasing for storing element and not to find it each time https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html#Elements
```
 cy.get(buttonSelector).as('button').eq(0).should('be.enabled');
 cy.get('@button').eq(1).should('be.enabled');
```
and so on